### PR TITLE
Add Linting for development

### PR DESCRIPTION
**Pull request by JcdeA**
add eslint & prettier & more rules to lint code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+  "version": "1.0.0",
+  "workspaces": [
+    "./lib/*",
+    "./dev/*"
+  ],
+  "scripts": {
+    "start": "concurrently --kill-others \"cd lib/destack && npm run dev\" \"cd dev/nextjs-project && npm run dev\"",
+    "prepare": "husky install"
+  },
+  "prettier": {
```
Comment (LiveDuo) [inline package.json:12]: Can we remove this as we have a `.prettierrc.js`?

File: lib/destack/server/utils/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
-const formParse = (form, req) => new Promise((resolve, reject) => {
+import fs from 'fs'
+import {NextApiRequest} from 'next'
+const formParse = (form: import("formidable/Formidable"), req:NextApiRequest) => new Promise<Object>((resolve, reject) => {
```
Comment (LiveDuo) [inline lib/destack/server/utils/index.ts:3]: Out of curiosity, `import("formidable/Formidable")` is inline:
```js
const formParse = (form: import("formidable/Formidable"), req:NextApiRequest) =&gt; new Promise&lt;Object&gt;((resolve, reject) =&gt; {Write<br class="Apple-interchange-newline">
```

does it not work if imported like so?
```js
import IncomingForm  from 'formidable/Formidable'
```

File: lib/destack/src/lib/css/index.ts [hunk 3]
```diff
@@ -18,9 +18,10 @@ const appendTailwindCss = (newEditor, setCssLoaded) => {
 }
 
 const appendCustomCss = () => {
-    document.querySelector("html").style.height = '100%'
-    document.querySelector("body").style.height = '100%'
-    document.querySelector("#__next").style.height = '100%'
+    document.querySelector("html")!.style.height = '100%'
+    document.querySelector("body")!.style.height= '100%'
+    let next: HTMLElement = document.querySelector("#__next")!
```
Comment (LiveDuo) [inline lib/destack/src/lib/css/index.ts:23]: Did you encounter any problems with?
```ts
document.querySelector("#__next")!.style.height = '100%'
```

I'm thinking the variable here can be a`const`:
`let next: HTMLElement ` → `const next: HTMLElement `

**Review by LiveDuo — CHANGES_REQUESTED** [review]
Hey, nice work with sorting out prettier & eslint and husky to better maintain the quality of the code and the project in general.

Just a few comments and will be ready to merge!

File: package.json (hunk 1 above)
Comment (JcdeA) [reply package.json:12]: Yep!

File: lib/destack/server/utils/index.ts (hunk 2 above)
Comment (JcdeA) [reply lib/destack/server/utils/index.ts:3]: Probably, It was vscode auto import that did it. I'll test it out in a minute.

File: lib/destack/src/lib/css/index.ts (hunk 3 above)
Comment (JcdeA) [reply lib/destack/src/lib/css/index.ts:23]: The problem is that  the normal html elements (such as body, div) have the style property, but others don't.

There probably is a way to do this cleanly, I'll look into this.

Comment (JcdeA) [reply lib/destack/src/lib/css/index.ts:23]: @LiveDuo t

> const next: HTMLElement
this has been fixed in another commit I think

**Comment by JcdeA** [comment]
shoot  I  added the wrong commits to the linting branch

**Pull request closed without merging**